backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/ApiUserResolver.java

```diff
@@ -22,15 +28,35 @@
 	private ApiUserService _userService;
 	private OrganizationService _organizationService;
 
-	public ApiUserResolver(OrganizationService os, ApiUserService users, OrganizationInitializingService initer) {
+	public ApiUserResolver(OrganizationService os, ApiUserService users) {
 		_organizationService = os;
 		_userService = users;
 	}
 
 	public User getWhoami() {
 		ApiUser currentUser = _userService.getCurrentUser();
-		Optional<CurrentOrganizationRoles> currentOrgRoles = _organizationService.getCurrentOrganizationRoles();
-		Boolean isAdmin = _userService.isAdminUser(currentUser);
+		Optional<OrganizationRoles> currentOrgRoles = _organizationService.getCurrentOrganizationRoles();
+		Boolean isAdmin = _userService.isAdmin(currentUser);
         return new User(currentUser, currentOrgRoles, isAdmin);
 	}
+
+	public List<User> getUsers() {
```

Comment:
yeah i couldn't find a good way to push this down the stack without pushing the User api model object down the stack as well, which smelled bad. *shrug*

Reply:
It may become more clear what's possible after other refactors are done and/or as other new code lands. No sense trying to move things around just for the sake of moving them, though.